frontend/src/components/NgoOverview.tsx

```diff
@@ -11,35 +11,277 @@ type NgoOverviewProps = {
+                                .filter(ngo => ngo
+                                    .sdg
+                                    .toLowerCase()
+                                    .includes(sdgFilter))
+                                .filter(ngo => ngo
+                                    .name
+                                    .toLowerCase()
+                                    .includes(searchName.toLowerCase()))
+                                .map(obj => <NgoCard ngo={obj}/>)}
+                        </div>
                     </Tab>
                     <Tab eventKey="europe" title="Europe">
-                        <p>europe</p>
+                        <div className={"NgoHeader"}>
+                            <h1>NGOs in Europe</h1>
+                            <input type={"text"}
+                                   value={searchName}
+                                   placeholder={"Type to search ..."}
+                                   onChange={(event:ChangeEvent<HTMLInputElement>) => setSearchName(event.target.value)}/>
+                            <div className={"filterSdg"}>
+                                <b>FILTER BY SDG </b>
+                                <ButtonGroup>
+                                    {sdgOptions.map((radio, idx) => (
+                                        <ToggleButton
+                                            key={idx}
+                                            id={`radio-${idx}`}
+                                            type="radio"
+                                            size="sm"
+                                            variant={'outline-success'}
+                                            name="radio"
+                                            value={radio.value}
+                                            checked={sdgFilter === radio.value}
+                                            onChange={(e) => setSdgFilter(e.currentTarget.value)}
+                                        >{radio.name}</ToggleButton>
+                                    ))}
+                                </ButtonGroup>
+                            </div>
+                        </div>
+                        <div className={"NgoBoard"}>
+                            {ngos.filter(ngo => ngo
+                                .region
+                                .toLowerCase()
+                                .includes("europe"))
+                                .filter(ngo => ngo
+                                    .sdg
+                                    .toLowerCase()
+                                    .includes(sdgFilter))
+                                .filter(ngo => ngo
+                                    .name
+                                    .toLowerCase()
+                                    .includes(searchName.toLowerCase()))
+                                .map(obj => <NgoCard ngo={obj}/>)}
+                        </div>
                     </Tab>
```

Comment:
SonarCloud is right: You coult create a component with this content to not repeat this code so often. "europe" or "oceania" could be in the props as `type ComponentProps = {name: string}`.